File: aoc2025/java/src/test/java/com/peeekay/aoc2025/java/Day03Test.java
package com.peeekay.aoc2025.java;

import com.peeekay.aocCommon.AOCPuzzle;
import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;

public class Day03Test {
    final AOCPuzzle day = new Day03(true);

    @Test
    void part1() {
        Assertions.assertEquals(357L, day.part1());
    }

    @Test
    void part2() {
        Assertions.assertEquals(3121910778619L, day.part2());
    }
}

File: test/com/peeekay/aoc2022/kotlin/Day03Test.kt
package com.peeekay.aoc2022.kotlin

import org.junit.jupiter.api.Assertions.assertEquals
import org.junit.jupiter.api.DisplayName
import org.junit.jupiter.api.Test

@DisplayName("Day 03")
internal class Day03Test {

    @Test
    fun `Part1 with test data`() {
        val day = Day03(isTest = true)
        val answer = day.partOne()

        assertEquals(157, answer)
    }

    @Test
    fun `Part2 with test data`() {
        val day = Day03(isTest = true)
        val answer = day.partTwo()

        assertEquals(70, answer)
    }
}